addons/hr_timesheet/views/project_portal_templates.xml

```diff
@@ -45,8 +45,10 @@
 
     <template id="portal_tasks_list_inherit" inherit_id="project.portal_tasks_list" name="Portal: My Tasks with Timesheets">
         <xpath expr="//t[@t-call='portal.portal_table']" position="inside">
-            <t t-set="timesheet_ids" t-value="task.sudo().timesheet_ids"/>
-            <t t-set="is_uom_day" t-value="timesheet_ids._is_timesheet_encode_uom_day()"/>
+            <t t-if="task">
+                <t t-set="timesheet_ids" t-value="task.sudo().timesheet_ids"/>
+                <t t-set="is_uom_day" t-value="timesheet_ids._is_timesheet_encode_uom_day()"/>
+            </t>
```

Comment:
Why not changing the xpath to add the t-set in the right place? It is a bit weird for me to add this t-if if it is just because those elements are in the wrong place. 🤔 

What do you think? 🤔